mailbox/zoo-seq-provider/pom.xml

```diff
@@ -59,14 +46,15 @@
             <artifactId>apache-james-mailbox-store</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.netflix.curator</groupId>
-            <artifactId>curator-recipes</artifactId>
-            <version>${curator.version}</version>
-        </dependency>
-        <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>13.0</version><!--$NO-MVN-MAN-VER$-->
+            <version>13.0</version>
+            <!--$NO-MVN-MAN-VER$-->
```

Comment:
That looks like some rules of the sort plugin. Is it worth fixing? After information is still here, on the right place.